Site/ASAB Maestro/Descriptors/elasticsearch.yaml

```diff
@@ -71,6 +71,11 @@ nginx:
     - proxy_cache_valid 200 10s
     - proxy_ignore_headers Cache-Control Expires Set-Cookie
 
+    # Increase buffer size to fit large Authorization headers
+    - proxy_buffer_size         32k
+    - proxy_buffers           4 64k
```

Comment:
https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffers

NGINX has 8 buffers ... `proxy_buffers 8 64k` is correct.

Reply:
https://github.com/TeskaLabs/asab-maestro-library/pull/94